.github/matrix.json

```
@@ -1,10 +1,12 @@
 {
     "swift_version": [
+        "6.0",
         "5.10",
         "5.9",
         "5.8"
     ],
     "swiftlint_supported_version": {
+        "6.0": "main",
         "5.10": "main",
         "5.9": "main",
```

Now that SwiftLint requires Swift 6 on Linux, I wonder if it would work with these changes...

```suggestion
        "6.0": "main",
        "5.10": "0.57.0",
        "5.9": "0.57.0",
```

---

I have already changed them in #65 and not rebased yet 😅